### ISSUE TITLE
Compilation fails on Android with latest Flutter 'master' build

### ISSUE THREAD

**Issue by doppio** [0maru/twitter_login#92]
After updating Flutter to latest master, Android builds fail due to errors with this package:

```
e: [.../.pub-cache/hosted/pub.dartlang.org/twitter_login-4.1.0/android/src/main/kotlin/com/maru/twitter_login/TwitterLoginPlugin.kt](): (21, 8):
'public open fun onNewIntent(Intent): Boolean defined in com.maru.twitter_login.TwitterLoginPlugin' clashes with 'protected*protected and package* open fun onNewIntent(Intent): Unit defined in io.flutter.embedding.android.FlutterActivity': return types are incompatible

e: [.../.pub-cache/hosted/pub.dartlang.org/twitter_login-4.1.0/android/src/main/kotlin/com/maru/twitter_login/TwitterLoginPlugin.kt](): (21, 14):
Cannot infer visibility for 'fun onNewIntent(p0: Intent): Boolean'. Please specify it explicitly

e: [.../.pub-cache/hosted/pub.dartlang.org/twitter_login-4.1.0/android/src/main/kotlin/com/maru/twitter_login/TwitterLoginPlugin.kt](): (76, 5):
'onNewIntent' overrides nothing
```

**Comment by 0maru** [comment]
@doppio 
Error confirmed. Thanks.
I will fix it within a few days.

```
twitter_login: 4.1.0

flutter 
[✓] Flutter (Channel master, 2.13.0-0.0.pre.469, on macOS 12.2.1 21D62 darwin-x64, locale ja-JP)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 13.3)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2020.3)
[✓] Android Studio (version 4.2)
[✓] IntelliJ IDEA Ultimate Edition (version 2021.3.1)
[✓] IntelliJ IDEA Ultimate Edition (version 2021.3)
[✓] VS Code (version 1.66.1)
[✓] Connected device (3 available)
[✓] HTTP Host Availability
```

**Comment by 0maru** [comment]
Fixed in #97
